org.goldendict.desktop.yml

```diff
@@ -0,0 +1,146 @@
+app-id: org.goldendict.desktop
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+base-version: '5.15'
+command: goldendict
+# separate-locales: false
+base: io.qt.qtwebkit.BaseApp
+finish-args:
+  - --share=ipc
+  # GoldenDict use lots of X11 functions and it currently cannot work
+  # natively on Wayland.
+  - --socket=x11
+  - --device=dri # hardware acceleration
+  - --socket=pulseaudio # videos/audios
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+rename-appdata-file: goldendict.appdata.xml
+rename-desktop-file: goldendict.desktop
+modules:
+  - name: opencc
+    buildsystem: cmake-ninja
+    build-options:
+      env:
+        CC: 'ccache gcc'
+        CXX: 'ccache g++'
+    sources:
+      - type: git
+        url: https://github.com/BYVoid/OpenCC.git
+        tag: ver.1.1.1
+        x-checker-data:
+          type: anitya
+          project-id: 7230
+          tag-template: ver.$version
+
+  - name: lzo2
+    config-opts:
+      - '--enable-shared'
+      - '--disable-static'
+    cleanup:
+      - /include
+      - /share/doc
+      - '*.la'
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: 'https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz'
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+        x-checker-data:
+          type: anitya
+          project-id: 1868
+          url-template: http://www.oberhumer.com/opensource/lzo/download/lzo-$version.tar.gz
+
+  - name: eb
+    config-opts:
+      - '--enable-shared'
+      - '--disable-static'
+    cleanup:
+      - /bin
+      - /share/doc
+    sources:
+      - type: git
+        url: https://salsa.debian.org/debian/eb.git
+        tag: debian/4.4.3-13
+        x-checker-data:
+          type: anitya
+          project-id: 155542
+          tag-template: debian/$version
+      # Update config.guess & config.sub to allow builds for aarch64 and others
```

Comment:
Just do something like this: https://github.com/flathub/org.glimpse_editor.Glimpse/blob/cd50b8421b2719438aa2b0518a15b1254a16c5a0/org.glimpse_editor.Glimpse.json#L52

Reply:
That's elegant